target/linux/lantiq/patches-4.19/4050-MIPS-lantiq-EBU-set_buscon_params.patch

```diff
@@ -0,0 +1,32 @@
+--- a/arch/mips/include/asm/mach-lantiq/xway/lantiq_soc.h
++++ b/arch/mips/include/asm/mach-lantiq/xway/lantiq_soc.h
+@@ -87,6 +87,8 @@ extern __iomem void *ltq_cgu_membase;
+ #define LTQ_EBU_PCC_ISTAT	0x00A0
+ #define LTQ_EBU_BUSCON1		0x0064
+ #define LTQ_EBU_ADDRSEL1	0x0024
++#define LTQ_EBU_BUSCON2		0x0068
++#define LTQ_EBU_ADDRSEL2	0x0028
+ #define EBU_WRDIS		0x80000000
+ 
+ /* WDT */
+--- a/arch/mips/lantiq/xway/sysctrl.c
++++ b/arch/mips/lantiq/xway/sysctrl.c
+@@ -147,7 +147,9 @@
+ 
+ static void __iomem *pmu_membase;
+ void __iomem *ltq_cgu_membase;
++
+ void __iomem *ltq_ebu_membase;
++EXPORT_SYMBOL(ltq_ebu_membase);
+ 
+ static u32 ifccr = CGU_IFCCR;
+ static u32 pcicr = CGU_PCICR;
+@@ -474,7 +476,7 @@
+ 	if (!pmu_membase || !ltq_cgu_membase || !ltq_ebu_membase)
+ 		panic("Failed to remap core resources");
+ 
+-	/* make sure to unprotect the memory region where flash is located */
++	/* make sure to unprotect the memory region where NOR flash is located */
```

Comment:
I think this comment change can be dropped.

Reply:
No idea, better i leave it

Reply:
This is addressing a cosmetic issue in a totally different area of the file compared to the functional changes above. I do not see a reason for keeping it.